### PR TITLE
add auth oauth resources to manifests

### PR DESCRIPTION
**Pull request by sallyom**
This PR adds the authentication and oauth CRDs
@enj is this what you'll need? or no CRs?

### CODE REVIEW

**Comment by enj** [comment]
I accidentally lost the run level changes when I rebased so you will want to use the 658607fddb875a9c37b6313a26460e321e7b066f naming format.

Otherwise this is fine for now, we will want to figure out the final resting place for the CRDs and CRs at some point before beta.

File: manifests/0000_09_cluster-osin-operator_08_auth_cr.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v1
+items:
```
Comment (enj) [inline manifests/0000_09_cluster-osin-operator_08_auth_cr.yaml]: uh just realized this is a list instead of a single item, same with other CR

Comment (sallyom) [reply manifests/0000_09_cluster-osin-operator_08_auth_cr.yaml]: : ) /hold

Comment (sallyom) [reply manifests/0000_09_cluster-osin-operator_08_auth_cr.yaml]: yup, cluster is much happier now,  i guess it wasn't the flaky CI, it was my flakiness, time for a break :)

**Pull request merged**